app/mailers/notifications.rb

```diff
@@ -90,13 +90,22 @@ def consultation_deadline_passed(consultation)
 
 private
 
+  def is_production?
+    /production/i.match?(GovukAdminTemplate.environment_label)
```

Comment:
If `environment_style` is the more "computery" version of this text, should we not do a simpler string comparison against that instead of titleizing it and then doing a case insensitive regex match against it?

Reply:
Also wondering whether `is_production?` provides any more useful a name than `production?` would.